packages/client/src/shared/utils/error-boundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component, ErrorInfo, ReactNode } from 'react'
```

Comment:
suggestion: new jsx transform has been added to react v17. And as of version 17, there's actually just no need to import React anymore. You can remove import React